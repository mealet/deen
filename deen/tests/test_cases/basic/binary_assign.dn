fn main() i32 {
  // integer
  let int = 1;
  println!("{}", int);

  int += 10;
  println!("{}", int);

  int += 10;
  println!("{}", int);

  // floating
  let float = 1.0;
  println!("{}", float);

  float += 0.1;
  println!("{}", float);

  float += 0.1;
  println!("{}", float);
  
  return 0;
}
