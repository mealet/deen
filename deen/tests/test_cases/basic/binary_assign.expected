1
11
21
1.000000
1.100000
1.200000
